### PR TITLE
[PVR][guiinfo][Estuary] PVR info dialog, recordings window: cleanup and extend

### PR DESCRIPTION
**Pull request by ksooo**
guiinfo: Add LISTITEM_EXPIRATION_DATE and LISTITEM_EXPIRATION_TIME 
Estuary: PVR info dialog & recordings window: Reorder some items and add expriration date/time for recordings:

PVR info dialog (before):
screenshot000

PVR info dialog (after):
screenshot003

Recordings window (before):
screenshot001

Recordings window (after):
screenshot004

This has been runtime tested on macOS, latest master.

@ronie, phil65 fyi

### CODE REVIEW

File: xbmc/GUIInfoManager.cpp [hunk 1]
```diff
@@ -3732,6 +3732,16 @@ const infomap container_str[]  = {{ "property",         CONTAINER_PROPERTY },
...
+///   \table_row3{   <b>`ListItem.ExpirationDate`</b>,
+///                  \anchor ListItem_ExpirationDate
+///                  _string_,
+///     Expiration date of the selected item in a container, empty string if not supported
+///   }
+///   \table_row3{   <b>`ListItem.ExpirationTime`</b>,
+///                  \anchor ListItem_ExpirationTime
+///                  _string_,
+///     Expiration time of the selected item in a container, empty string if not supported
+///   }
```
Comment (AlwinEsch) [inline xbmc/GUIInfoManager.cpp]: You need to add a **`\`** before **`,`** inside text lines! Also seen on `ListItem.EpgEventTitle` missing (may be also somewhere else).

```
Expiration time of the selected item in a container\, empty string if not supported
```

Thats a stupid bad thing on doxygen by use of my table style.

**Review by AlwinEsch — COMMENTED** [review]
How it is shown if addon not support expire date for recordings?

File: xbmc/GUIInfoManager.cpp (hunk 1 above)
Comment (ksooo) [inline xbmc/GUIInfoManager.cpp]: okay, will change

**Comment by da-anda** [comment]
could you move the "genre" and "expires" info to a second column in the PVR info dialog? Currently there is so much lost space on the right handside of it. Alternatively, have you tried moving the plot on the right? These are just two quick ideas. I also just thought about moving the "meta data" below the thumbnail. But this dialog might in general need a bit more love from our skinners. @phil65 ?

**Comment by ksooo** [comment]
I agree, but due to lack of knowledge on my end I leave this to our skinners, in a follow-up PR.

File: xbmc/GUIInfoManager.cpp (hunk 1 above)
Comment (ksooo) [inline xbmc/GUIInfoManager.cpp]: done

**Comment by ksooo** [comment]
> How it is shown if addon not support expire date for recordings?

 :-) The "Expiration: foo" line is not present, then.

**Review by AlwinEsch — APPROVED** [review]

**Comment by da-anda** [comment]
@ksooo fair point. I hope one of our skinners will give this dialog some love. Btw, is the channel info below the recording title really important? I think the episode title ("Steg mit Aussicht") might be better suited there, and the channel name moved down to the meta data. Just an idea and no hard feelings.

**Comment by ksooo** [comment]
jenkins build this please

**Comment by ksooo** [comment]
@da-anda better now?

screenshot001

**Comment by da-anda** [comment]
yep, better IMO. But I would have done it like "Sender: Das Erste HD" to match the other meta data. Up to you if you want to adjust it that way. And thanks much for maintaining and improving PVR.

**Comment by ksooo** [comment]
I had it exactly like you suggested, but channel name imo is special and important; it needs to be highlighted from other less important properties.

**Comment by da-anda** [comment]
> but channel name imo is special and important; it needs to be highlighted from other less important properties

is it? That's probably subject of personal preferences and recording habits. I primarily care for the show I recorded and not on which channel it was on. But again, no hard feelings, fine with whatever you prefer.

Btw, because it just came to my mind. Might be nice to also have a "delete" button in the OSD as well as "find similar" and maybe even "edit timer" (in case it's from a recurring timer rule, if we store that info somewhere). Just some ideas for the future, as the "play" button is kinda lost down there.

**Comment by ksooo** [comment]
@phil65 are the skin changes okay?

**Comment by phil65** [comment]
yup, looks fine.

**Review by phil65 — APPROVED** [review]

**Comment by ksooo** [comment]
jenkins build this please

**Comment by ksooo** [comment]
jenkins errors are unrelated

**Pull request merged**